tools/dataMigration/migrateUsers.ts

```diff
@@ -54,7 +53,9 @@ export const migrateUsers = async (props: { client: BaseProtocol }): Promise<voi
                    end as status,
                u_l.profile_picture_file,
                u_l.position,
-               u_l.email
+               u_l.email,
+               -- Avoid possible conflicts with ids from legacy database
+               (u_l.id + 1000000) as id
```

Comment:
I really don't like this .. sorry 😣

Reply:
we don't have to do it
should we still include the id in the migration though?